### ISSUE TITLE
Provide more reliable outgoing mail (i.e. preconfigure TKL to use a remote relay)

### ISSUE THREAD

**Issue by lirazsiri** [turnkeylinux/tracker#482]
https://aws.amazon.com/marketplace/review/product-reviews/ref=dtl_pop_customer_reviews?ie=UTF8&asin=B00E8NLFKO

Bugzilla needs to be able to generate emails, and the TurnKey solution is not setup to properly send emails hooked up to Amazon's SES email system (which is the only way you are sending emails outside of AWS without them being flat out rejected by inbound mail servers.) 

I understand that TurnKey would not extensively document DNS DKIM records that are required, and the other requirements that AWS imposes in order to get setup with SES. 

However, their out-of-the-box Bugzilla setup was not equipped with the TLS SNMP module and adjustments to the Bugzilla script that are necessary to support TLS. AWS SES only works with TLS (or STARTTLS). 

Fortunately, there is a lot of online info on how to get Bugzilla working with TLS. This site has the exact how-to that worked for me: 

http://www.dawood.in/bugzilla-alerts-using-gmail/ 
##

**Comment by lirazsiri** [comment]
Another related complaint:

https://aws.amazon.com/marketplace/review/product-review-comments/?ie=UTF8&asin=B007I8OOJ0&appAction=&appActionToken=1TYAklheosVcoTeaokga5a2fQz0j3D&reviewId=_1RRUC8FC8Q7S0

**Comment by JedMeister** [comment]
For the record; the perl module `Email::Send::SMTP::TLS` is [packaged in Debian](https://packages.debian.org/jessie/libemail-sender-transport-smtp-tls-perl) so can be easily installed with:
````
apt-get update
apt-get install libemail-sender-transport-smtp-tls-perl
````

**Comment by JedMeister** [comment]
@OnGle has also done some email related documentation on the TurnKey docs: https://www.turnkeylinux.org/docs/email-via-remote-smtp

**Comment by JedMeister** [comment]
FWIW the original issue from the AWS Marketplace is actually a specific bugzilla issue. It has been resolved for v14.1 (see #454 ).

However a superior general solution to the issue of reliable email delivery is still required. However, unfortunately it won't make it into v14.1...

**Comment by JedMeister** [comment]
This functionality has now been included in confconsole as of v1.0.0+ It should be released any day now...
